### ISSUE TITLE
Provide a generated to-do list based on failed atomic tests and critical errors

### ISSUE THREAD

**Issue by jenoe** [w3c/silver#305]
## Reasoning

Provide incentives for continuous improvements.

## Proposal

An additional result of an accessibility audit based on WCAG3 should be an automatically generated to-do list based on all failed atomic tests (automatic and manual) as well as on all critical errors.

Even though this might not be part of core WCAG3, the catalogue of provided tools and instruments should include or at least support such a mechanism.

This could help website owners with a clear path to improve the current state of accessibility conformance in order to start a continuous process instead of a static one-time evaluation.

The task items should be as specific and atomic as possible, e.g. refer to a specific image that misses an appropriate alt-text.

**Comment by jpascalides** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by fstrr** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines \(WCAG\) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
